### DISCUSSION TITLE
GitHub Mobile Notification doesn't indicate what changed.

### DISCUSSION THREAD

**Discussion by davidwengier** [community/community#13462] [Mobile] [answered]
I got a notification about an issue, and it shows me the title, a snippet of the last comment, and the status icon. I clicked the notification and started reading the comment, before realising it was from 3 years ago. The issue had just been closed.

Would be good if the comment line was replaced with "Closed" or the status image indicated that it was the thing that had changed.

![image](https://user-images.githubusercontent.com/754264/160030284-f7cce34b-0fd4-402c-9f1b-56b5e7259e35.png)

P.S. I submitted this feedback via the "Report Feedback" link on the app, and it was routed to the support team, who said [it was better suited to the product team](https://support.github.com/ticket/personal/0/1559327), so they closed it. So my other feedback might be that the feedback system within the app could be a lot better!

**Comment by aconsuegra** [comment, marked as answer]
Hi @davidwengier, thanks for reaching out! I agree that the notification subtitle is misleading, we should try to fix it. I created an internal issue to track it, and I hope we can find a good solution soon. Keep the feedback coming!

Regarding the feedback flow, noted. That's not a great flow.
